src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/dm/DocumentManagementUploadService.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.dm;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticatorFactory;
+import uk.gov.hmcts.reform.document.DocumentUploadClientApi;
+import uk.gov.hmcts.reform.document.domain.UploadResponse;
+import uk.gov.hmcts.reform.document.utils.InMemoryMultipartFile;
+
+import static java.util.Collections.singletonList;
+
+@Service
+public class DocumentManagementUploadService {
+
+    private static final Logger log = LoggerFactory.getLogger(DocumentManagementUploadService.class);
+
+    @Autowired
+    private DocumentUploadClientApi documentUploadClientApi;
+
+    private final CcdAuthenticatorFactory ccdAuthenticatorFactory;
+
+    private static final String FILES_NAME = "files";
+
+    DocumentManagementUploadService(CcdAuthenticatorFactory ccdAuthenticatorFactory) {
```

Comment:
can you use constructor injection for the `DocumentUploadClientApi` ? no point mixing and matching

Reply:
Updated the class to use constructor injection for `DocumentUploadClientApi`